Description: The "wheel" group is not existing on debian, check the "sudo" one instead
Author: Laurent Bigonville <bigon@debian.org>
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=760722
Forwarded: not-needed

Index: gnome-control-center-3.22.2/panels/common/gnome-control-center.rules
===================================================================
--- gnome-control-center-3.22.2.orig/panels/common/gnome-control-center.rules
+++ gnome-control-center-3.22.2/panels/common/gnome-control-center.rules
@@ -6,7 +6,7 @@ polkit.addRule(function(action, subject)
 	     action.id == "org.gnome.controlcenter.datetime.configure") &&
 	    subject.local &&
 	    subject.active &&
-	    subject.isInGroup ("wheel")) {
+	    subject.isInGroup ("sudo")) {
 		    return polkit.Result.YES;
 	    }
 });
